.circleci/config.yml

```diff
@@ -40,9 +40,9 @@ defaults:
       cp .circleci/prepare-commit-msg .git/hooks/prepare-commit-msg
       chmod +x .git/hooks/prepare-commit-msg
 
-  requires_build_core: &requires_build_core
+  requires_build_elements_core: &requires_build_elements_core
     requires:
-      - build_core
+      - build_elements_core
```

Comment:
Do we expect any packages that don't start with `elements-`? If not, we can drop `elements` from the job names and add the prefix where we use it instead. That should also significantly reduce the diff for this file.